assets/app.js

```diff
@@ -105,10 +95,19 @@
             repo.language +
           '</p>'
         );
+        if ($('body').attr('data-prerendered') === 'data-prerendered') {
+          $link.find('.repo__info').append(
+            '<br>' +
+            'Updated ' +
+            prettyDate(repo.pushed_at)
```

Comment:
can we include this label in the html to prevent the jumping when the js loads ? We could do something like `$link.find('.repo__updated').html( prettyDate(repo.pushed_at) );`